src/guest_webapp/nginx.conf

```diff
@@ -0,0 +1,9 @@
+server {
```

Comment:
Some additional options to investigate:

Enable compression
```
gzip on;
gzip_types text/html application/javascript application/json text/css;
```

Enable Caching
```
location ~* \.(?:css|js|jpg|svg)$ {
    expires 30d;
    add_header Cache-Control "public";
}
```
Reference:
https://sdickinson.com/nginx-config-for-single-page-applications/